src/pages/docs/en/quickstart-guides/node.md

```diff
@@ -54,4 +56,4 @@ micros:
 - `include` specifies which directory to include in the final app package, everything else will be ignored. In our case we set it to the directory of our build output
 - `run` specifies the command to start your Micro. In our case running the `index.js` file in our `build` directory using `node`
```

Comment:
aren't people having issues with `node index.js`?

Reply:
There have been no problems with this in the tests I've done with a few different deployments.